RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -272,6 +349,37 @@ class RNUnifiedContacts: NSObject {
 
     return emailAddresses
   }
+  
+  func transformNumber(number: NSDictionary) -> CNLabeledValue {
+    var label = String()
+    if (number["label"] as! String == "home") {
```

Comment:
@joshuapinter as it turns out, seems like iOS allows empty labels. It doesn't throw any exceptions etc. I guess they built it that way so that it could seamlessly integrate with Google Contacts and other services which don't necessarily support labels on phone numbers and emails. 

Perhaps we should allow blank values as well and leave it up to the developers to decide if they wish to implement it in their apps or not. What are your thoughts?